index.bs

```diff
@@ -12261,6 +12252,20 @@ defined in section [[#platformobjectset]].
     1.  Perform steps 3-11 of the [=default [[Set]] internal method=].
 </div>
 
+<h4 id="platformobjectsetprototypeof">Platform object \[[SetPrototypeOf]] method</h4>
+
+<div algorithm="to invoke the internal [[SetPrototypeOf]] method of platform objects">
+
+    The internal \[[SetPrototypeOf]] method of every [=platform object=]
+    that implements an [=interface=] with the [{{Global}}] or [{{PrimaryGlobal}}]
+    [=extended attribute=] must execute the same algorithm must be executed as is defined for the
```

Comment:
"must execute the same algorithm must be executed" has too many "musts".  Probably should be:

  ... must execute the same algorithm as is defined for ...